examples/influxdb_data_model.ipynb

```diff
@@ -136,8 +137,8 @@
    "metadata": {},
    "source": [
     "The rationale for this mapping is the following:\n",
-    "1. `ci_id`, `run_id` and `squash_id` are sequential ids. `ci_url` and `run_id_url` are different for each run. The reason for adding them as InfluxDB fields is to [reduce InfluxDB series cardinality](https://docs.influxdata.com/influxdb/v1.7/concepts/schema_and_data_layout/#discouraged-schema-design) (DM-18342) and we also want to display these information in the dashboards.\n",
-    "2. We define a common datamodel mapping variables from different execution environments example `ci_id` and `run_id` to `run_id`, `ci_url` and `run_id_url` to `run_url`, `ci_dataset` and `dataset` to `dataset`. The corresponding values can be filtered by the `env_name` tag. \n",
+    "1. The reason for mapping `ci_id`, `run_id`, `squash_id`, `ci_url` and `run_id_url` to InfluxDB fields is to [reduce InfluxDB series cardinality](https://docs.influxdata.com/influxdb/v1.7/concepts/schema_and_data_layout/#discouraged-schema-design) `ci_id`, `run_id` and `squash_id` are sequential ids and `ci_url` and `run_id_url` are also different for each run (DM-18342). We also want to display these information in dashboards so they m.\n",
```

Comment:
There's a missing period after "reduce InfluxDB series cardinality". The end of the line seems cut off.